pkg/api/types.go

```diff
@@ -398,6 +402,21 @@ type NFSVolumeSource struct {
 	ReadOnly bool `json:"readOnly,omitempty"`
 }
 
+// GlusterfsVolumeSource represents a Glusterfs Mount that lasts the lifetime of a pod
+type GlusterfsVolumeSource struct {
+	// Required: hosts is the endpoint name that details Glusterfs topology
+	Hosts string `json:"hosts"`
```

Comment:
Let's not call it Hosts anymore, since that doesn't immediately make people think of a Kubernetes Endpoint name,